account/recovery.go

```diff
@@ -17,6 +17,24 @@ var (
 	DefaultAccountKeyWindow uint32 = 500
 )
 
+// GetAuctioneerData returns the auctioner data for a given environment.
+func GetAuctioneerData(network string) (string, uint32) {
+	var auctioneerKey string
+	var fstBlock uint32
+
+	switch network {
+	case "mainnet":
+		auctioneerKey = "028e87bdd134238f8347f845d9ecc827b843d0d1e2" +
```

Comment:
Define these as nice constants at the top of the file? Together with the batch key in the next commit.